src/sdk-reference/js/6/auth/refresh-token/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: sdk.html.hbs
+title: refreshToken
+description: Refresh an authentication token
+---
+
+# refreshToken
+
+Refreshes a valid, non-expired authentication token.
+
+If this action is successful, all further requests emitted by this SDK instance will use the refreshed authentication token.
+
+<br/>
+
+```javascript
+refreshToken ([options])
+```
+
+<br/>
+
+| Arguments    | Type    | Description |
+|--------------|---------|-------------|
+| `options`  | <pre>object</pre> | Query options |
+
+
+### options
+
+Additional query options
+
+| Property     | Type<br/>(default)    | Description   |
+| -------------- | --------- | ------------- |
+| `expiresIn` | <pre>string</pre> | Expiration time in [ms library](https://www.npmjs.com/package/ms) format. (e.g. `2h`) |
```

Comment:
What is the default value?

Reply:
There isn't one per se, it's configured in the kuzzlerc file.

Reply:
Maybe you can add a note or something to specify that so developers knows that a default value is configurable

Reply:
I didn't want to, because it meant I had to change all `login` documentations for all SDK, as well as bring those documentation to today's format. :expressionless: 

But you're right. So... thank you for noticing. I guess. :expressionless: 

It's done. :+1: